### PR TITLE
Update SECURITY.md

### PR DESCRIPTION
**Pull request by umer936**
I'm not sure that this is correct. Found from: https://github.com/cakephp/cakephp/security . I'm basing it off what it used to have and https://endoflife.date/cakephp 

It used to have some "Release of x.x" instead of "# months" which might be better wording 

Please edit this PR lol

### CODE REVIEW

**Comment by markstory** [comment]
Looks correct to me.

File: .github/SECURITY.md [hunk 1]
```diff
@@ -6,9 +6,11 @@ We support fixing security issues on the following releases:
 
 | Version | Supported          | Security fixes until
 | ------- | ------------------ | -------------------- 
-| 4.4     | :white_check_mark: | 24 Months after the release of 5.0
-| 4.3     | :white_check_mark: | Release of 5.0
-| 4.2     | :white_check_mark: | Release of 4.5
+| 5.0     | :white_check_mark: | 24 Months after the release of 5.0
+| 4.5     | :white_check_mark: | 24 Months after the release of 5.0
+| 4.4     | :x:                | 24 Months after the release of 5.0
```
Comment (othercorey) [inline .github/SECURITY.md:11]: This should be "No longer supported" I think

File: .github/SECURITY.md [hunk 2]
```diff
@@ -6,9 +6,11 @@ We support fixing security issues on the following releases:
 
 | Version | Supported          | Security fixes until
 | ------- | ------------------ | -------------------- 
-| 4.4     | :white_check_mark: | 24 Months after the release of 5.0
-| 4.3     | :white_check_mark: | Release of 5.0
-| 4.2     | :white_check_mark: | Release of 4.5
+| 5.0     | :white_check_mark: | 24 Months after the release of 5.0
```
Comment (othercorey) [inline .github/SECURITY.md:9]: This should be TBD after the release of 6.0 I think.

File: .github/SECURITY.md (hunk 1 above)
Comment (markstory) [reply .github/SECURITY.md:11]: I'm fine doing security releases on 4.4 for a while longer. We don't have many usually.

Comment (othercorey) [reply .github/SECURITY.md:11]: Yea, you will do a one-off for a real issue, but not sure how we state that. Leave it to you.

**Pull request merged**